src/patcher.js

```diff
@@ -15,6 +28,19 @@ class PatchedBrowserWindow extends BrowserWindow {
       opts.webPreferences.nodeIntegration = true;
     }
 
+    if (transparentWindow) {
+      console.log('transparent')
+      opts.transparent = true
+      opts.backgroundColor = '#00000000'
+      opts.frame = false
+    }
+
+    if (experimentalWebPlatform) {
+      if (!opts.webPreferences) opts.webPreferences = {}
+      opts.webPreferences.experimentalFeatures = true
+    }
```

Comment:
this whole if block (and the one for transparentWindow) can just be moved to the existing if-statement, i.e.  

```js
if (opts.webPreferences && opts.webPreferences.preload) {
  global.originalPreload = opts.webPreferences.preload;
  opts.webPreferences.preload = join(__dirname, 'preload.js');
  opts.webPreferences.nodeIntegration = true;

  if (transparentWindow) {
    opts.transparent = true;
    opts.frame = false;
    opts.backgroundColor = '#00000000';
  }

  if (experimentalWebPlatform) {
    opts.webPreferences.experimentalFeatures = true;
  }
}
```